pkgs/tools/networking/ngrok-3/update.sh

```diff
@@ -10,7 +10,7 @@ get_download_info() {
          https://update.equinox.io/check \
          'Accept:application/json; q=1; version=1; charset=utf-8' \
          'Content-Type:application/json; charset=utf-8' \
-         app_id=app_goVRodbMVm \
+         app_id=app_c3U4eZcDbjV \
```

Comment:
This file holds a reference to `ngrok-2` at the bottom.

Reply:
Oh whoops, yep. Just tested it out and it works now... plus there's an update. Done.